test/Interop/Cxx/function/default-arguments-module-interface.swift

```diff
@@ -0,0 +1,31 @@
+// RUN: %target-swift-ide-test -print-module -module-to-print=DefaultArguments -I %S/Inputs -source-filename=x -cxx-interoperability-mode=upcoming-swift | %FileCheck %s
+
+// CHECK: func isZero(_ value: Int32 = cxxDefaultArg) -> Bool
```

Comment:
is it possible to map some default arguments to literal constants instead, to present a better module interface?

Reply:
Good point, I was initially thinking of reusing the parsing logic in `ImportMacro.cpp` to import simple default values such as `nullptr` without synthesizing new functions. I didn't have time to implement that, but I think this would be a great future improvement.